conversations.yaml

```diff
@@ -0,0 +1,618 @@
+        default: conversations.sc4.medallia.com
+        description:
+          The Medallia Experience Cloud hostname used to access
+          Medallia Conversations' API gateway.
+  - url: 'https://{adapter-api-gateway-hostname}'
+    variables:
+      adapter-api-gateway-hostname:
+        default: adapter-gateway.example.com
+        description:
+          The endpoint used to make requests from Medallia Conversations
+          to the channel adapter.
+paths:
+  /:
+    servers:
+      - url: 'https://{adapter-api-gateway-hostname}'
+        variables:
+          adapter-api-gateway-hostname:
+            default: adapter-gateway.example.com
+            description:
+              The endpoint used to make requests from Medallia
+              Conversations to the channel adapter.
+    post:
+      tags:
+        - Channel Adapter Services
+      description:
+        The API endpoint implemented by the Channel Adapter to accept
+        messages from Medallia Conversations.
+      parameters:
+        - in: query
+          name: access_token
+          schema:
+            type: string
+          required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: '#/components/schemas/OutboundMessageRequest'
+                - $ref: '#/components/schemas/EndOfConversationSignal'
+      responses:
+        200:
+          description: 
+            A successful response includes the customer and message IDs.
+          content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/OutboundMessageResponse'
+  /{customer-id}:
+    servers:
+      - url: 'https://{adapter-api-gateway-hostname}'
+        variables:
+          adapter-api-gateway-hostname:
+            default: adapter-gateway.example.com
+            description:
+              The endpoint used to make requests from Medallia
+              Conversations to the channel adapter.
+    get:
+      tags:
+        - Channel Adapter Services
+      description:
+        Returns a customer's profile data for use in Medallia
+        Conversations' reporting interfaces. All fields returned 
+        by the adapter are optional. The adapter returns whatever
+        it can, and it may return an empty response.
+      parameters:
+        - in: path
+          name: customer-id
+          schema:
+            type: string
+          required: true
+        - in: query
+          name: access_token
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description:
+            The customer's profile data. If the customer id provided 
+            is not found, an empty response should be returned.
+          content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/CustomerProfileData'
+  /apis/v0/channels/{channel-guid}/messages:
```

Comment:
Need to add 401 - Unauthorized

Reply:
Need to add 404 - Not Found when the customer id is not found by the adapter.  Add a description to explain this.